features/performance/README.md

```diff
@@ -17,3 +19,18 @@ oc -n openshift-machine-config-operator wait machineconfigpools worker --for con
 - get pod UID by running `oc get pods -o custom-columns=NAME:.metadata.name,UID:.metadata.uid,HOST_IP:.status.hostIP`
 - now you can find the specific cgroup under the host where the pod is running and check hugepages limit `cat /sys/fs/cgroup/hugetlb/kubepods.slice/kubepods-pod<pod_uid>.slice/hugetlb.1GB.limit_in_bytes`
 - under the pod you can check the huge pages mount `mount | grep hugepages`
+
+## Topology Manager
+
+To enable the topology manager, you should:
+
+- enable the topology manager feature gate `oc apply -f manifests/05-fg-latency-sensetive.yaml`
+- wait for workers update
+
+```bash
+oc -n openshift-machine-config-operator wait machineconfigpools worker --for condition=Updating --timeout=1800s
+oc -n openshift-machine-config-operator wait machineconfigpools worker --for condition=Updated --timeout=1800s
+```
+
+- update the kubelet configuration `oc apply -f manifests/05-kubeletconfig-worker-rt.yaml`
+- wait for workers update
```

Comment:
@cwilkers Will it be enough to add `cat /etc/kubernetes/kubelet.conf` from the worker node, to see that all parameters are here? Or are you wanting validation that topology manager is working(checking cgroups)?

Reply:
@cynepco3hahue I am thinking more along the lines of experimental verification like with the HugePages support--running a pod and demonstrating CPU pinning, but that may be work for another PR. If that's out of scope, I think what is in here now lgtm.

Reply:
@cwilkers I likes the idea to check it normally:
- use topology manager with `single-numa-node
` or `best-effort`
- use CPU manager with `static` policy
- use some device plugin that allocates PCI device(I think it can be GPU or SR-IOV)
- run the pod with the resource request for 1 CPU and 1 PCI device
- verify that pod allocated resources placed on the same NUMA node

But I do not have resources to check it normally(no host), so I will add a basic check for the kubelet config under the README

By the way, huge pages still not supported by the topology manager.

And thanks for the review:)